test/client/js/pie-item.test.js

```diff
@@ -42,10 +45,7 @@ describe('PieItem', function() {
 
   beforeEach(function() {
     this.container.innerHTML = `<pie-item></pie-item>`;
-    return customElements.whenDefined('pie-item')
-      .then(_ => {
-        this.el = this.container.querySelector('pie-item');
-      });
+    this.el = this.container.querySelector('pie-item');
```

Comment:
You're right, thanks! It looks like `customElements.define()` is an asynchronous method, but it doesn't return a promise, because that would be too easy. Instead, you have to call `customElements.whenDefined()` with the same element name you used in `define()`, which then does return a promise. Gotta love the web.

Fixed.